diff -Nur -x '*.orig' -x '*.rej' e-smith-flexbackup-1.9.0/root/etc/e-smith/templates/etc/flexbackup.conf/10prune_tree mezzanine_patched_e-smith-flexbackup-1.9.0/root/etc/e-smith/templates/etc/flexbackup.conf/10prune_tree
--- e-smith-flexbackup-1.9.0/root/etc/e-smith/templates/etc/flexbackup.conf/10prune_tree	2005-11-19 11:00:43.752629578 +1100
+++ mezzanine_patched_e-smith-flexbackup-1.9.0/root/etc/e-smith/templates/etc/flexbackup.conf/10prune_tree	2005-11-19 11:00:28.648307658 +1100
@@ -1,10 +1,10 @@
 {
     my @prunelist = split(',', $flexbackup{Prune} || '');
-    if (scalar @prunelist)
-    {
-	$OUT .= '$prune';
-	$OUT .= "{'/'} = '";
-	$OUT .= join(' ', @prunelist);
-	$OUT .= "';";
-    }
+    
+    return "" unless (scalar @prunelist);
+
+    $OUT = '$prune';
+    $OUT .= "{'/'} = '";
+    $OUT .= join(' ', @prunelist);
+    $OUT .= "';";
 }
