diff -Nur -x '*.orig' -x '*.rej' e-smith-flexbackup-1.9.0/root/etc/e-smith/events/actions/tape-restore-flexbackup mezzanine_patched_e-smith-flexbackup-1.9.0/root/etc/e-smith/events/actions/tape-restore-flexbackup
--- e-smith-flexbackup-1.9.0/root/etc/e-smith/events/actions/tape-restore-flexbackup	2006-01-08 20:05:24.000000000 -0500
+++ mezzanine_patched_e-smith-flexbackup-1.9.0/root/etc/e-smith/events/actions/tape-restore-flexbackup	2006-01-08 19:57:45.000000000 -0500
@@ -59,6 +59,7 @@
 sub ProbeTape ($);
 sub RewindTape ($);
 sub FSFTape ($$);
+sub DetermineBlocksize($);
 sub CreateFileList($);
 sub ExtractFiles($$);
 sub NotifyAdmin ($$);
@@ -85,8 +86,11 @@
 &ProbeTape($cfg::device);
 &RewindTape($cfg::device);
 &FSFTape($cfg::device, $tapefile);
+my $tape_blocksize = &DetermineBlocksize($cfg::device);
+&RewindTape($cfg::device);
+&FSFTape($cfg::device, $tapefile);
 &CreateFileList($tempfile);
-&ExtractFiles($filesystem, $tempfile);
+&ExtractFiles($filesystem, $tempfile, $tape_blocksize);
 
 ###########################################################################
 
@@ -186,6 +190,22 @@
     }
 }
 
+sub DetermineBlocksize($)
+{
+    my $device = shift;
+    use File::Temp;
+    use File::stat;
+
+    my $dir = File::Temp::tempdir( CLEANUP => 1 );
+    my $file = "$dir/xx";
+
+    system("mt", "-f", "$device", "fsf", "1");
+    system("dd", "if=$device", "bs=1M", "count=1", "of=$file");
+
+    my $st = stat($file) or die "No $file: $!";
+    return (($st->size)/1024);
+}
+
 sub DetermineBackupType
 {
     my $device = shift;
@@ -224,10 +244,11 @@
     close FILE;
 }
 
-sub ExtractFiles ($$)
+sub ExtractFiles ($$$)
 {
     my $filesystem = shift;
     my $filelist = shift;
+    my $blocksize = shift;
 
     unless (chdir $filesystem)
     {
@@ -250,6 +271,7 @@
     system(
 	    "/usr/bin/flexbackup",
 	    "-extract",
+	    "-d", "blocksize=$blocksize",
 	    "-flist",
 	    $filelist
 	);
